src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/LoggingRedactionOptions.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET8_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Compliance.Classification;
+using Microsoft.Extensions.Http.Diagnostics;
+using Microsoft.Shared.DiagnosticIds;
+
+namespace Microsoft.AspNetCore.Diagnostics.Logging;
+
+/// <summary>
+/// Top-level model for redacting incoming HTTP requests and their corresponding responses.
+/// </summary>
+[Experimental(diagnosticId: Experiments.HttpLogging, UrlFormat = Experiments.UrlFormat)]
+public class LoggingRedactionOptions
```

Comment:
For all the properties, the default value should be documented in a `<value>` block, rather than in `<remarks>`

Reply:
This type shoudl be using option validation code gen to ensure non-nullable fields are not forced to null, and enum values are in range, etc.

Reply:
There aren't any fields that can be forced to null. How does the enum validation work? I don't see any attributes for that.